core/sdk/src/api/firebolt/fb_capabilities.rs

```diff
@@ -84,6 +111,23 @@ impl FireboltCap {
         None
     }
 
+    pub fn parse_long(cap: String) -> Option<FireboltCap> {
+        let pattern = r"^xrn:firebolt:capability:([a-z0-9\\-]+)((:[a-z0-9\\-]+)?)$";
```

Comment:
there is an existing parses method can we update that?

Reply:
Updated